package.json

```diff
@@ -13,11 +13,14 @@
     "seed": "node database/seeding.js"
   },
   "dependencies": {
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
     "express": "^4.17.1",
     "mongodb": "^3.6.3",
     "mongoose": "^5.11.13",
     "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "react-dom": "^17.0.1",
+    "styled-components": "^5.2.1"
```

Comment:
Just a heads up if you haven't already. Look into the styled components plug-in for babel.